### ISSUE TITLE
[TorrentBuilder] crash when creating a large torrent with POSIX wrapper

### ISSUE THREAD

**Issue by proninyaroslav** [aldenml/libtorrent4j#48]
Hello. Crash happens if I create a torrent with a fairly large number of files (as an example, I created a torrent of the `/storage/emulated/0` dir of my device, which contains more than 100 files). Without POSIX wrapper, a torrent is successfully created, but the following exception occurs with enabled POSIX wrapper. Important note: I use the default POSIX wrapper, that is, I call only `super` methods in my child class, for example `super.stat()`.
```
java.io.IOException: Try again
        at org.libtorrent4j.TorrentBuilder.generate(TorrentBuilder.java:507)
```
If I create a torrent with fewer files (for example 1 or 10) then everything happens fine.

**Comment by aldenml** [comment]
Hi, do you have a working project that I can use to test and debug this and the other reported issues?

**Comment by proninyaroslav** [comment]
I think that I can prepare a special project for you without extra code, so that it is easier to debug it.

**Comment by aldenml** [comment]
if it's a well created open source project, that might work

**Comment by proninyaroslav** [comment]
Ok, please see `2.0-dev-frozen` branch of the app (it's a bit messy because work in progress): https://github.com/proninyaroslav/libretorrent/tree/2.0-dev-frozen.

The main code that can help you:
 - [core/LibTorrentSafAdapter.java](https://github.com/proninyaroslav/libretorrent/blob/2.0-dev-frozen/app/src/main/java/org/proninyaroslav/libretorrent/core/LibTorrentSafAdapter.java) - a POSIX wrapper class.
 - [core/SafFileSystem.java](https://github.com/proninyaroslav/libretorrent/blob/2.0-dev-frozen/app/src/main/java/org/proninyaroslav/libretorrent/core/SafFileSystem.java) - used to work with SAF, for example in POSIX wrapper.
 - [core/FileSystemFacade.java](https://github.com/proninyaroslav/libretorrent/blob/2.0-dev-frozen/app/src/main/java/org/proninyaroslav/libretorrent/core/FileSystemFacade.java) - all filesystem operations (`SafFileSystem` and classic `File`).
 - [core/TorrentBuilder.java](https://github.com/proninyaroslav/libretorrent/blob/2.0-dev-frozen/app/src/main/java/org/proninyaroslav/libretorrent/core/TorrentBuilder.java) - a wrapper for `org.libtorrent4.TorrentBuilder` class.
 - [viewmodel/CreateTorrentViewModel.java](https://github.com/proninyaroslav/libretorrent/blob/2.0-dev-frozen/app/src/main/java/org/proninyaroslav/libretorrent/viewmodel/CreateTorrentViewModel.java) - a view model logic that uses `TorrentBuilder` wrapper.
 - [core/TorrentSessionImpl.java](https://github.com/proninyaroslav/libretorrent/blob/2.0-dev-frozen/app/src/main/java/org/proninyaroslav/libretorrent/core/TorrentSessionImpl.java) - implementation of `org.libtorrent4.SessionManager`. POSIX wrapper installation occurs in `init()` method.

**Comment by proninyaroslav** [comment]
`1.2.2.0` still has this problem.

**Comment by proninyaroslav** [comment]
It seems to be fixed in `1.3.0-alpha-5`.
